rivershared/cmd/update-mod-version/main.go

```diff
@@ -26,6 +27,10 @@ func main() {
 }
 
 func run() error {
+	// Allows secondary River repositories to make use of this command by
+	// specifying their own prefix.
+	packagePrefix := cmp.Or(os.Getenv("PACKAGE_PREFIX"), "github.com/riverqueue/river")
```

Comment:
Could just make this a required input and skip having a default value? Your call.

Reply:
Yeah, good point. Can't hurt to be explicit about this. Updated the command and added `PATH_PREFIX` in this project's `Makefile`.